src/view-utilities.js

```diff
@@ -114,16 +112,72 @@ var viewUtilities = function (cy, options) {
     }
 
     return eles
-            .removeClass("highlighted")
-            .removeClass("unhighlighted")
+    .removeClass("highlighted")
+    .removeClass("unhighlighted")
             .removeData("highlighted"); // TODO check if remove data is needed here
-  };
+          };
 
   // Indicates if the ele is highlighted
   instance.isHighlighted = function (ele) {
     return ele.is(".highlighted:visible") ? true : false;
   };
 
+
+  //Zoom selected Nodes
+  instance.zoom = function ( eles){
+    eles.unselect();
+    cy.animate({
+      fit: {
+        eles: eles,
+        padding: 20
+      }
+    }, {
+      duration: 1000
+    });  
+    return eles;
+  };
+
+  instance.marqueeZoom = function( canvas){
```

Comment:
Unnecessary parameter, please remove it. Also, we don't put single space before parameter names, please remove it here and in similar cases.